Formula/ponyc.rb

```diff
@@ -13,17 +12,27 @@ class Ponyc < Formula
     sha256 "007c43a75741ecd4c7c96c9fc67d816ebc845c0aac94687c4e2cbb1ee959ebfb" => :mavericks
   end
 
-  depends_on "llvm" => "with-rtti"
+  depends_on "llvm"
   depends_on "libressl"
   depends_on "pcre2"
   needs :cxx11
 
   def install
     ENV.cxx11
-    system "make", "install", "config=release", "destdir=#{prefix}", "verbose=1"
+    ENV["LLVM_CONFIG"]="#{Formula["llvm"].bin}/llvm-config"
```

Comment:
All of these `.bin` or `.lib` references in this PR should be `.opt_bin` and `.opt_lib`.